Automattic/wp-calypso
# Importers: Simplify Importers screens

#### Changes proposed in this Pull Request

* Remove "Start import" buttons and importer descriptions in favor of clickable cards.
* Remove `FormattedHeader` and subheading in favor of a more concise `SectionHeader`
* Simplify copy on individual importer screens.
* Update font sizes and styles for clearer information hierarchy.
* Adds form labels to text inputs for accessibility

(I know this looks like a big PR, but it's mostly style changes and string changes. Normally I would break this into multiple PRs, but to work faster and avoid merge conflicts, I kept everything in a single branch. Hope this doesn't make it too hard to review. 🤞 )

**Before**

<img width="895" alt="Screen Shot 2019-10-21 at 4 53 44 PM" src="https://user-images.githubusercontent.com/2124984/67243060-c73a5680-f424-11e9-9551-7a877a91949c.png">

**After**

<img width="1161" alt="Screen Shot 2019-10-21 at 5 03 17 PM" src="https://user-images.githubusercontent.com/2124984/67243068-cc97a100-f424-11e9-94ed-0ca4e2b9eae3.png">


#### Testing instructions

* Switch to this PR and view a simple WP.com site
* Go to Tools -> Import
* Check each importer to make sure it works as expected
* Make sure there are no visual bugs on desktop or mobile web

Addresses Issues #36729 #36732 #36735 #36733 and #36612
